### PR TITLE
Update function-python-ai-textsummarize README

### PR DESCRIPTION
**Pull request by v-jiaodi**
Update function-python-ai-textsummarize README to use `azd init` and `azd up` instead of `azd up --template` according to https://github.com/Azure/azure-dev/issues/1769.

@jongio, @rajeshkamal5050 for notification.

### CODE REVIEW

**Comment by rajeshkamal5050** [comment]
@jongio @paulyuk can you review and approve?

**Comment by rajeshkamal5050** [comment]
@paulyuk can you merge it?

File: README.md [hunk 1]
```diff
@@ -81,7 +81,24 @@ You will see AI analysis happen in the Terminal standard out.  The analysis will
 
 The easiest way to deploy this app is using the [Azure Dev CLI aka AZD](https://aka.ms/azd).  If you open this repo in GitHub CodeSpaces the AZD tooling is already preinstalled.
 
-To provision and deploy:
+1. Run the following command to initialize the project. 
+
+```bash
+azd init --template https://github.com/Azure-Samples/function-python-ai-textsummarize
```
Comment (paulyuk) [inline README.md:87]: We should not do an init template in a git repo.  What we have today is more correct.

**Comment by paulyuk** [comment]
Hi - when the user starts from a Git repo like this we should embrace the git flow (clone, restore, run, azd up) and not "initialize from template" which i would want to do if I didnt have a repo.

**Pull request closed without merging**